src/uct/base/uct_component.c

```diff
@@ -105,6 +105,12 @@ ucs_status_t uct_config_read(uct_config_bundle_t **bundle,
     uct_config_bundle_t *config_bundle;
     ucs_status_t status;
 
+    if ((config_table == NULL) || (config_size == 0)) {
```

Comment:
why is this needed in this PR?

Reply:
gtest segfaults without this change

Reply:
yes but why? when is the config_table null?

Reply:
iface config table is NULL now for rdmacm component

Reply:
why is there a flow which creates iface/md on rdmacm component?
it should not have any iface/md resources

Reply:
this is how UCT tests works now, I planned to remove them together with sockcm
anyway I think better do not segfault on reading empty config if we user don't have other way to detect this

Reply:
`config_table == NULL` is considered invalid param for this function..
is it possible to disable/skip/remove the UCT tests which cause this?

Reply:
Ok I'll will cherry-pick tests clean up which I prepared for next PR with sockcm

Reply:
tests clean up doesn't help, fixed in MD config read